src/plugins/controls/public/options_list/components/options_list_popover_suggestions.tsx

```diff
@@ -159,6 +159,13 @@ export const OptionsListPopoverSuggestions = ({
       <div ref={listRef}>
         <EuiSelectable
           options={selectableOptions}
+          renderOption={(option) => {
+            return (
+              <EuiHighlight search={option.key === 'exists-option' ? '' : searchString.value}>
+                {option.label}
+              </EuiHighlight>
```

Comment:
I don't think it makes sense for the "Exists" option to be highlighted, since it always shows up when searching anyway: